### PR TITLE
update README.md

### PR DESCRIPTION
**Pull request by 0Armaan025**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,80 +68,24 @@ docker run --detach -it --name Admidio \
...
-- [ramsey/uuid](https://github.com/ramsey/uuid): A PHP library for generating and working with UUIDs.
-- [SecurImage](https://www.phpcaptcha.org/): PHP CAPTCHA script for generating complex images and CAPTCHA codes to protect forms from spam and abuse.
-- [Select2](https://select2.github.io/): jQuery replacement for select boxes
-- [Smarty](https://www.smarty.net/): PHP template engine
-- [TCPDF](https://tcpdf.org/): Open Source PHP class for generating PDF documents.
-- [zxcvbn](https://github.com/dropbox/zxcvbn): Add password strength indicator
-- [zxcvbn-php](https://github.com/bjeavons/zxcvbn-php): Test password strength in the backend
-
-Copyright (c) 2004 - 2023 The Admidio Team
+If you appreciate Admidio and our project, consider [making a donation](https://www
```
Comment (Fasse) [inline README.md:91]: The text is truncated. Could you please complete the description.

**Review by Fasse — CHANGES_REQUESTED** [review]

**Comment by 0Armaan025** [comment]
Done please check

File: README.md [hunk 2]
```diff
@@ -85,80 +68,25 @@ docker run --detach -it --name Admidio \
...
-- [ramsey/uuid](https://github.com/ramsey/uuid): A PHP library for generating and working with UUIDs.
-- [SecurImage](https://www.phpcaptcha.org/): PHP CAPTCHA script for generating complex images and CAPTCHA codes to protect forms from spam and abuse.
-- [Select2](https://select2.github.io/): jQuery replacement for select boxes
-- [Smarty](https://www.smarty.net/): PHP template engine
-- [TCPDF](https://tcpdf.org/): Open Source PHP class for generating PDF documents.
-- [zxcvbn](https://github.com/dropbox/zxcvbn): Add password strength indicator
-- [zxcvbn-php](https://github.com/bjeavons/zxcvbn-php): Test password strength in the backend
-
-Copyright (c) 2004 - 2023 The Admidio Team
+If you appreciate Admidio , consider donating please
```
Comment (Fasse) [inline README.md:91]: Hmm, I'm not satisfied with this part. Maybe something like this:

> If you appreciate Admidio, consider making a [donation](https://www.admidio.org/donate.php).
> Thanks for contributing!

**Review by Fasse — CHANGES_REQUESTED** [review]

**Comment by Fasse** [comment]
@0Armaan025 I would like to change the donating part. Please have a look.

**Comment by Fasse** [comment]
@0Armaan025 Thanks for contributing :)

**Pull request merged**